releases/index.html

```diff
@@ -106,6 +106,13 @@ <h1>Releases</h1>
                 <h2 id="repo">GenePattern Notebook Repository</h2>
                 <p>Release updates for the <a href="https://notebook.genepattern.org" target="_blank">GenePattern Notebook Repository</a>.</p>
 
+                <h3>GenePattern Notebook Repository 19.10 <em style="font-size: 0.8em; color: gray;">(October 28, 2019)</em></h3>
+                <ul>
+                    <li>Pinned public notebook tags now display an optional description.</li>
+                    <li>Updated to JupyterHub 1.0.</li>
+                    <li>Updated to latest versions of genepattern-notebook, nbtools and jupyter-wysiwyg.</li>
```

Comment:
these should be linked to the relevant sections in this doc for ease of navigation

Reply:
Same thing here - Michael's was fine with leaving the release notes layout as is, so long as we linked to the sections of the document in the line that reads "Updated to latest versions" (line 113)